ogr/ogrsf_frmts/pg/ogrpgdatasource.cpp

```diff
@@ -2355,6 +2376,62 @@ OGRLayer *OGRPGDataSource::GetLayer(int iLayer)
         return papoLayers[iLayer];
 }
 
+/************************************************************************/
+/*                             FindSchema()                             */
+/************************************************************************/
+
+// Check that the schema exist. If there is a single match in a case
+// insensitive way, use it. Otherwise error out if th match is not exact.
```

Comment:
```suggestion
// Check that the schema exists. If there is a single match in a case
// insensitive way, use it. Otherwise error out if the match is not exact.
```

Reply:
done